march_hardware_builder/src/robots/march4.yaml

```diff
@@ -40,39 +40,39 @@ march4:
         temperatureges: {slaveIndex: 1, byteOffset: 0}
         imotioncube:
           slaveIndex: 5
-          encoder: {resolution: 17, minPositionIU: 2053, maxPositionIU: 45617, zeroPositionIU: 10261, safetyMarginRad: 0.05}
+          encoder: {resolution: 17, minPositionIU: 2053, maxPositionIU: 45617, zeroPositionIU: 9314, safetyMarginRad: 0.05}
```

Comment:
This is quite the difference, is this in line with what we witnessed during the training?